input_dictionary.json

```diff
@@ -461,6 +459,130 @@
+		"problem_parameters": [
+			{
+				"id": "n",
+				"type": "number",
+				"display_label": "Number of columns",
+				"placeholder": "Insert parameter",
+				"tooltip": "Number of columns of the input matrix"
+			},
+          	{
+				"id": "m",
+				"type": "number",
+				"display_label": "Number of rows",
+				"placeholder": "Insert parameter",
+				"tooltip": "Number of rows of the input matrix"
+			},
+           {
+				"id": "q",
+				"type": "number",
+				"display_label": "Field size",
+				"placeholder": "Insert parameter",
+                "tooltip": "An integer of the form p^x for some integer x, where p is prime number.  This value indicates the number of elements in the underlying field"
+           },
+
+          {
+				"id": "ell",
+				"type": "number",
+				"display_label": "No. of rows in the kernel",
+                "placeholder": "Insert value",
+                "default_value": 1,
+                "tooltip": "Number of rows of the matrix whose permutation should lie in the kernel"
+          },
+          {
+				"id": "use_parity_row",
+                "type": "switch",
+				"display_label": "Use parity row",
+                "default_value": false,
+                "tooltip": "Enables trick of appending extra (all one) row to the matrix, i.e., m -> m+1"
+          },
+          {
+                "id": "memory_bound",
+                "type": "number",
+                "direction": "column",
+                "display_label": "Memory limit",
+                "placeholder": "Insert value",
+                "caption": "Leave empty if no limit is desired",
+                "tooltip": "Log2 of the maximum number of bits of memory available"
+        },
+        {
+              "id": "nsolutions",
+              "type": "number",
+              "direction": "column",
+              "display_label": "Number of solutions",
+              "placeholder": "Insert value",
+              "tooltip": "Log2 of number of existing solutions of which one has to be found"
+        }
+        ],
+		"estimator_parameters": [
+        {
+              "id": "bit_complexities",
+              "type": "switch",
+              "display_label": "Bit complexities",
+              "default_value": true,
+              "tooltip": "Show complexities as count of bit operations. If false, show number of elementary operations"
+        },
+        {
+              "id": "included_algorithms",
+              "type": "multiple_selector",
+              "direction": "column",
+              "display_label": "Included algorithms",
+              "tooltip": "Algorithms to include for optimization",
+              "default_value": [],
+              "excluded_algorithms": [],
+              "options": [],
+              "dependencies": []
+        },
+        {
+              "id": "include_tildeo",
```

Comment:
even though no tilde o is implemented we want to show the switch in the frontend?

Reply:
We shouldn't, it might cause confusion.